build.gradle

```diff
@@ -31,8 +31,11 @@ dependencies {
     // GSON
     compile group: 'com.google.code.gson', name: 'gson', version: '2.8.0'
     // JSONP
-    compile group: 'javax.json', name: 'javax.json-api', version: '1.0'
-    compile group: 'org.glassfish', name: 'javax.json', version: '1.0.4'
+    compile group: 'javax.json', name: 'javax.json-api', version: '1.1.2'
```

Comment:
this actually doesn't compile (tests fail). the new version has a new "default" method in the reader interface without a default implementation and johnzon  1.10 doesn't implement it.

Reply:
When I checked it, johnzon implemented the default method of JsonReader. However, Johnson's test uses Genson as a JsonReader. Genson's JsonReader has not been implemented yet, so the test fails.
When JsonReader is used, Gsonon gets JsonProvider from `JsonProvider.provider()` in johnzon's MapperBuilder, and Genson's JsonProvider is returned.
In order to be a good test, JsonProvider (JsonReader) should also use johnzon's.
I will check this part and PR again.